src/backend/joanie/core/serializers.py

```diff
@@ -426,6 +426,7 @@ def update(self, instance, validated_data):
         The "failed" state can only be set by the LMSHandler.
         """
         validated_data.pop("course_run", None)
```

Comment:
This line can be removed as the field is readonly and only set in the "create" method...